src/supported-value.js

```diff
@@ -35,11 +63,64 @@ export default function supportedValue(property, value) {
 
   // It is a string or a number as a string like '1'.
   // We want only prefixable values here.
-  if (typeof value !== 'string' || !isNaN(parseInt(value, 10))) return value
+  if ((typeof value !== 'string' && !Array.isArray(value)) ||
+    !isNaN(parseInt(value, 10))) {
+    return value
+  }
+
+  let isDoubleArray = false
+  let isArray = false
+
+  if (Array.isArray(value)) {
```

Comment:
this entire block seems to be what https://github.com/cssinjs/jss/blob/master/src/utils/toCssValue.js is already doing or am I missing something? Can we use it?